src/bmi_cfe.c

```diff
@@ -152,8 +152,11 @@ Variable var_info[] = {
     { 84, "Schaake_adjusted_magic_constant_by_soil_type",   "double", 1},
     { 85, "a_Xinanjiang_inflection_point_parameter",        "double", 1},
     { 86, "b_Xinanjiang_shape_parameter",                   "double", 1},
-    { 87, "x_Xinanjiang_shape_parameter",                   "double", 1}
+    { 87, "x_Xinanjiang_shape_parameter",                   "double", 1},
     //---------------------------------------
+    /*Soil Freeze-thaw model paramaters*/
+    { 88, "soil__ice_fraction_schaake",			    "double", 1}, //should it be here?? I don't see other input/outut variables in this list- AJ
```

Comment:
These are variables within the soil_reservoir structure.  If they remain, should they be grouped with the soil_reservoir structure variables?

Reply:
Well, I think they should not stay here as they will be included in the list of "state variables" of SFT.

Reply:
DONE. Added these parameters to the state list and moved them to the soil reservoir struct list.